Clients/GUI/Pages/UserInfo.razor

```diff
@@ -3,162 +3,295 @@
-        </div>
-        <div class="form-group">
-            <label for="Email">Email:</label>
-            <InputText class="@InputStyleClass" disabled="true" @bind-Value="@user.Email" />
+}
+    <div class="wrapper">
+
+        <div align="center" class="h3" min-width:>
+            <h3><b>User Info</b></h3>
         </div>
-        <div class="form-group" style="@(IsPasswordBeingChanged ? "" : "height: 0; color: transparent")">
-            <label for="OldPassword">Enter old password: </label>
-            <InputText type="@(IsPasswordBeingChanged ? "password" : "hidden")" class="@InputStyleClass" disabled="@(!IsPasswordBeingChanged)" @bind-Value="@user.OldPassword" />
+        <nav class="navbar navbar-expand-sm" align ="center">
+            <ul class="navbar-nav m-auto text-lg-center" >
+                <li class="nav-item"><a class="nav-link" href="userinfo">Personal Information</a></li>
+                <li class="nav-item"><a class="nav-link" href="#">Transaction History</a></li>
+            </ul>
+        </nav>
+        <div class="row">
+            <div class="form-group col-sm-5" align ="center">
+                <EditForm Model="@user">
+                    <p><img src=@imgSrc style="max-width:200px; max-height:200px;" alt="Avatar"></p>
+                    <div class="@(IsEditModeDisabled ? "invisible" : "visible")">
+                        <br />
+                        <label for="files" disabled="@IsEditModeDisabled" class="btn btn-primary col-3">Select Image</label>
+                        <InputFile style="display: none" id="files"  accept=".jpg, .jpeg, .png" @onclick =" ChangeAvatarClickHandler" OnChange="ChangeAvatar" />
+                        <br />
+                        <button class="btn btn-secondary col-3" @onclick="Reset">Reset</button>
+                        <br />
+                    </div>
+                </EditForm>
+            </div>
+        <div class=" = col-sm-7">
+            <EditForm Model="@user">
+                <DataAnnotationsValidator />
+                <div class="form-group">
+                    <label for="Name">Name:</label>
+                    <InputText class="form-control col-3" @onclick="UserInfoChangeHandler" disabled="@IsEditModeDisabled" @bind-Value="user.FirstName" />
+                </div>
+                <div class="form-group">
+                    <label for="Surname">Surname:</label>
+                    <InputText class="form-control col-3" @onclick="UserInfoChangeHandler" disabled="@IsEditModeDisabled" @bind-Value="user.LastName" />
+                </div>
+                <div class="form-group">
+                    <label for="Birthday">Birthday:</label>
+                    <InputDate class="form-control col-3" @onclick="UserInfoChangeHandler" disabled="@IsEditModeDisabled" @bind-Value="@user.Birthday" />
+                </div>
+                <div class="form-group">
+                    <label for="Email">Email:</label>
```

Comment:
Email with average length does not fit here fully

Reply:
fixed